src/main/java/uk/gov/hmcts/ccd/domain/casestate/EnablingConditionSorter.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.ccd.domain.casestate;
+
+import java.util.Comparator;
+import java.util.List;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.domain.model.definition.EventPostStateDefinition;
+
+@Component
+public class EnablingConditionSorter {
+
+    public void sortEventPostStates(List<EventPostStateDefinition> eventPostStates) {
+        if (eventPostStates != null) {
```

Comment:
do we really need to add this null check here? is there any real chance eventPostStates can be null?

Reply:
Each event will have atleast one post state... I have removed this null check